usr/share/rear/output/PXE/default/820_copy_to_net.sh

```diff
@@ -15,6 +16,19 @@ case "$scheme" in
     (fish|ftp|ftps|hftp|http|https|sftp)
         LogPrint "Transferring PXE files to $OUTPUT_URL"
         for result_file in "${RESULT_FILES[@]}" ; do
+            # OUTPUT_URL for lftp transfers will look something like:
+            # "<fish|ftp|ftps|hftp|http|https|sftp>://<host_name>/<destination_directory>"
+            # Using `cut' for printing 4th field with "/" delimiter (.. | cut -d/ -f4-),
+            # will strip leading "<protocol>://<hostname>/" from OUTPUT_URL
+            # and by prefixing "/" we will get absolute path to destination directory.
+            dest_dir="/$(echo $OUTPUT_URL | cut -d/ -f4-)"
```

Comment:
Sorry - again I am wondering about `echo $OUTPUT_URL | cut -d/ -f4-`:
Doesn't our `url_path` function in lib/global-functions.sh do what you need?
See in particular the initial comment about all those `url_*` function that contains
```
# where a generic URI is of the form
# scheme:[//[user:password@]host[:port]][/]path[?query][#fragment]
# e.g. for BACKUP_URL=sshfs://user@host/G/rear/
# url_scheme = 'sshfs' , url_host = 'user@host' , url_hostname = 'host' , url_username = 'user' , url_path = '/G/rear/'
```

Reply:
Fixed via b47bbca12dc25e208731f7e2480389cc478bfa0d